src/look_transform.rs

```diff
@@ -21,7 +19,9 @@ pub struct LookTransformBundle {
 
 /// An eye and the target it's looking at. As a component, this can be modified in place of bevy's `Transform`, and the two will
 /// stay in sync.
-#[derive(Clone, Component, Copy, Debug)]
+#[derive(Component, Debug, PartialEq, Clone, Copy, Reflect)]
+#[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
```

Comment:
Shouldn't it be `feature = "serde"`?

Reply:
I honestly don't know since I was simply referencing [Bevy's Transform](https://github.com/bevyengine/bevy/blob/main/crates/bevy_transform/src/components/transform.rs#L39). I have never personally used serde since I am relatively new to Rust so I thought the Bevy Implementation couldn't be bad. I can change that if you want me to.

Reply:
You should read the documentation on cargo features: https://doc.rust-lang.org/cargo/reference/features.html

In particular, there is no "serialize" feature defined in the Cargo.toml for this crate, so your use of `cfg_attr` won't work. But the "serde" dependency is optional, so it can be referred to by name as a feature.

Reply:
Thanks for explaining that to me. Some parts of Rust are still new to me. I'll change it and test it out and push a new commit.

Reply:
Works great like you said it would by appending 
```rs
    let transform = LookTransform::new(eye, target, up);
    let str = serde_json::to_string(&transform).unwrap();
    info!("str = {}", str);
    let transform = serde_json::from_str::<LookTransform>(&str).unwrap();
    info!("transform = {:?}", transform);
```
in the above ``fn setup`` test code